library/errata_tool_product_version.py

```diff
@@ -39,9 +39,36 @@
        - Release key (eg. robosignatory uses this value)
        - You almost certainly never want to change this value.
      choices: [master, fedora, beta, test, none, redhatrelease, rhx,
-               redhatrelease2, redhatengsystems]
+               redhatrelease2, redhatbeta2, redhatengsystems, redhate2etesting,
+               redhatdevel, containerready1, redhatimabeta, redhatimarelease]
      required: false
      default: redhatrelease2
+   container_sig_key_name:
+     description:
+       - Release Key (eg. robosignatory uses this value).
+     choices: [master, fedora, beta, test, none, redhatrelease, rhx,
+               redhatrelease2, redhatbeta2, redhatengsystems, redhate2etesting,
+               redhatdevel, containerready1, redhatimabeta, redhatimarelease]
+     required: false
+     default:  redhatrelease2
+   use_quay_for_containers:
+     description:
+       - If enabled,container advisories using this product version
+        will be pushed to Quay.io instead of docker-pulp for live pushes.
```

Comment:
```suggestion
       - If enabled, the Errata Tool will instruct pub to push this product
         version's container advisories to quay.io instead of docker-pulp for
         live pushes.
```